lsp-methods.el

```diff
@@ -246,7 +246,7 @@ initialized. When set this turns off use of
   "Create notification body for method METHOD and parameters PARAMS."
   (inline-quote
     (progn (cl-check-type ,method string)
-      (list :jsonrpc "2.0" :method ,method :params ,params))))
+      (list :jsonrpc "2.0" :method ,method :params ,(or params (make-hash-table))))))
```

Comment:
parameters to a notification _can_ be null in some cases. See https://microsoft.github.io/language-server-protocol/specification#exit